app/src/speech-recognition-types.d.ts

```diff
@@ -0,0 +1,133 @@
+declare global {
```

Comment:
Does TypeScript not have built-in types for SpeechRecognition?

Reply:
I could not get it working without these; I did it before in JS, but the only examples I could find carried their own definitions like this. Probably it can be done nicer indeed.